.golangci.yml

```diff
@@ -0,0 +1,84 @@
+# v1.47.2
+# Please don't remove the first line. It is used in CI to determine the golangci version
+run:
+  deadline: 5m
+
+issues:
+  # Maximum issues count per one linter. Set to 0 to disable. Default is 50.
+  max-issues-per-linter: 0
+  # Maximum count of issues with the same text. Set to 0 to disable. Default is 3.
+  max-same-issues: 0
+
+  # We want to try and improve the comments in the k6 codebase, so individual
+  # non-golint items from the default exclusion list will gradually be added
+  # to the exclude-rules below
+  exclude-use-default: false
+
+  exclude-rules:
+    # Exclude duplicate code and function length and complexity checking in test
+    # files (due to common repeats and long functions in test code)
+    - path: _(test|gen)\.go
+      linters:
+        - cyclop
+        - dupl
+        - funlen
+        - gocognit
+        - lll
+    - linters:
+        - paralleltest # false positive: https://github.com/kunwardeep/paralleltest/issues/8.
+      text: "does not use range value in test Run"
+
+linters-settings:
+  cyclop:
+    max-complexity: 25
+  dupl:
+    threshold: 150
+  exhaustive:
+    default-signifies-exhaustive: true
+  funlen:
+    lines: 80
+    statements: 60
+  goconst:
+    min-len: 10
+    min-occurrences: 4
+  govet:
+    check-shadowing: true
+  maligned:
+    suggest-new: true
+  nolintlint:
+    # Disable to ensure that nolint directives don't have a leading space. Default is true.
+    allow-leading-space: false
+
+linters:
+  enable-all: true
+  disable:
+    - decorder
+    - execinquery
+    - exhaustivestruct
+    - exhaustruct
+    - gci
+    - gochecknoinits
+    - gocyclo # replaced by cyclop since it also calculates the package complexity
+    - godot
+    - godox
+    - goerr113 # most of the errors here are meant for humans
+    - goheader
+    - golint # this linter is deprecated
+    - gomnd
+    - gomodguard
+    - grouper
+    - interfacer # deprecated
+    - ireturn
+    - maintidx
+    - maligned # replaced by govet 'fieldalignment'
+    - nlreturn
+    - nonamedreturns
+    - nosnakecase
+    - scopelint # deprecated, replaced by exportloopref
+    - tagliatelle
+    - testpackage
+    - thelper
+    - varnamelen
+    - wrapcheck # a little bit too much for k6, maybe after https://github.com/tomarrell/wrapcheck/issues/2 is fixed
+    - wsl
```

Comment:
I think you need to update this list with the one from the current k6 `master` branch - maybe just copy the whole file ;)